DependencyInjection/DoctrineExtension.php

```diff
@@ -111,6 +111,23 @@ protected function dbalLoad(array $config, ContainerBuilder $container)
         $loader = new XmlFileLoader($container, new FileLocator(__DIR__.'/../Resources/config'));
         $loader->load('dbal.xml');
 
+       /*
+        *
+        * Convert some symfony-style naming on connection options so that we
+        * satisfy DBAL, which is expecting camelCasing.
+        *
+        */
+        $conversions = array('default_table_options' => 'defaultTableOptions');
```

Comment:
we already have code doing this. See ``getConnectionOptions``. Just add this option there